templates/DNADesign/Elemental/UserForms/Model/ElementForm.ss

```diff
@@ -0,0 +1,3 @@
+<div class="elementform-form">
```

Comment:
Re: discussion in Slack about class naming conventions (BEM) - we should call this `element__form`

Reply:
In https://github.com/silverstripe/silverstripe-elemental-blocks/pull/3 and via community Slack we decided on `[type]element__[element]` so we'd use `formelement__form` here